README.md

```diff
@@ -5,34 +5,83 @@
 PLUTO is a cross platform desktop application that can be used to connect to the deepmarket network.
 
 This project is in active development and is being maintained by the [team members](#team-members) listed below.
-If you would like to submit changes, please open a pull request.
+If you would like to submit changes, please open a pull request. Please read the [getting set up to develop locally](https://github.com/deepmarket/PLUTO/wiki/Setting-up-deepmarket's-api-backend-for-local-development) guide.
 
-**Note**: This project requires Python 3.6+
 
-#### Installing, Developing, and Testing:
+**Note**: This project requires Python 3.6. Make sure python3.6 is installed before attempting any of this!
 
-First you'll need to install the required dependencies
+[Installer](https://www.python.org/downloads/release/python-366/)
 
+To check version:
+
+```bash
+$ python3 --version
+```
+
+## Installing, Testing, Running:
+
+#### Installation
+
+First you'll need to install the required dependencies.  We use the `pipenv` package management tool to make managing dependencies easier.  For more info on the tool, see [here](https://docs.pipenv.org/en/latest/basics/).  To install:
 ```bash
-$ python -m pip install -r requirements.txt
+$ python3 -m pip install pipenv
 ```
 
-Next, you can read the [getting set up to develop locally](https://github.com/deepmarket/PLUTO/wiki/Setting-up-deepmarket's-api-backend-for-local-development) guide.
+Then to install dependencies:
+
+```bash
+$ pipenv install
+```
+
+If you receive an error related to the python binary, try:
+
+```bash
+$ pipenv install --python=$(which python3)
+```
+
+#### Runtime Environment
+
+Use `pipenv` to activate a virtual environment necessary for running this project.  
+
+```bash
+$ pipenv shell
+```
+
+#### Testing
 
 Testing is done using two libraries: Python's builtin `Unittest` for unit testing and `behave` for functional BDD tests.
 
-They can be ran independently
+They can be ran independently.  Make sure that the virtual environment has been activated before running these.
+
 ```bash
-$ python -m  unittest test/*.py
+$ python -m unittest discover ./src/unittest/python
+$ python -m behave ./src/integrationtest/python
+```
 
-$ python -m behave test/features
+Alternately, these can be run outside the virtualenv:
```

Comment:
It looks like these are backwards as using `pipenv run` should indeed run the tests within a `virtualenv`.

Reply:
yeah I guess I worded this confusingly.  What I meant is that running stuff with `pipenv run` doesn't require a virtualenv to be activated beforehand.  I'll make it a little more clear